apps/range-voting/app/components/VoteRow.js

```diff
@@ -62,7 +67,18 @@ class VoteRow extends React.Component {
     return (
       <TableRow>
         <StatusCell onClick={this.handleVoteClick}>
-          {open ? <Countdown end={endDate} /> : <VoteStatus vote={vote} />}
+          <div>
+            <div>
+              {open ? <Countdown end={endDate} /> : <VoteStatus vote={vote} />}
+            </div>
+            {!open && getVoteStatus(vote) === VOTE_STATUS_SUCCESSFUL &&
+              <div>
+                <Button mode="strong" wide onClick={this.handleExecuteVote}>
+                  Execute Vote
+                </Button>
+              </div>
+            }
+          </div>
```

Comment:
It seems the status is not being updated after the vote ends, until I open the sidepanel or refresh the app